iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -701,10 +733,21 @@ Inspector.prototype.switchItem = function (id) {
     this.update();
 }
 
+Inspector.prototype.preferredLanguages = function () {
+    var urlParams = new URLSearchParams(window.location.search);
+    if (urlParams.has("lang")) {
+        return [ urlParams.get("lang") ];
+    }
+    var langs = window.navigator.languages || [ window.navigator.language || window.navigator.userLanguage ] ;
```

Comment:
Can `language` and `userLanguage` both be null (null reference in the `.each` below), or does that never happen in practice for modern browsers?

Reply:
I suspect it's safe, but I've made it safer in 87ed933